src/container/styles.scss

```diff
@@ -61,11 +64,68 @@
   }
 }
 
+.with-side-media {
+  display: flex;
+  flex-direction: row;
+}
+
+.with-top-media {
+  display: flex;
+  flex-direction: column;
+}
+
+.content-wrapper {
+  flex: 1;
+  &-fit-height {
+    overflow: auto;
+  }
+  &-with-media {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+.media {
+  overflow: hidden;
+
+  // stylelint-disable-next-line @cloudscape-design/no-implicit-descendant, selector-max-type
+  img,
+  video,
+  picture {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    object-position: center;
+  }
+
+  &-top {
+    max-height: 80%;
+    border-top-left-radius: awsui.$border-radius-container;
+    border-top-right-radius: awsui.$border-radius-container;
+  }
+
+  &-side {
+    max-width: 66%;
+    border-top-left-radius: awsui.$border-radius-container;
+    border-bottom-left-radius: awsui.$border-radius-container;
+  }
+}
+
 .header {
   background-color: awsui.$color-background-container-header;
   border-top-left-radius: awsui.$border-radius-container;
   border-top-right-radius: awsui.$border-radius-container;
 
+  &.header-with-media {
+    background: none;
+    &:not(:empty) {
+      border-bottom: none;
+    }
+    &.content {
+      overflow: auto;
```

Comment:
Why is this needed? Why do we want containers with media to scroll their content?

Reply:
Removed!